sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java

```diff
@@ -10,6 +10,8 @@
  */
 public class KeyVaultLoadStoreParameter implements KeyStore.LoadStoreParameter {
 
+    private static final String DEFAULT_AAD_AUTHENTICATION_URL = "https://login.microsoftonline.com/";
```

Comment:
Should we also set the URL [here](https://github.com/chenrujun/azure-sdk-for-java/blob/2ad3072ff46f2163ddb878d2adfe909e20f9a3dc/sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultLoadStoreParameter.java#L62) as the default URL